components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/selection/GeckoSelectionActionDelegate.kt

```diff
@@ -58,8 +58,12 @@ open class GeckoSelectionActionDelegate(
     }
 
     override fun performAction(id: String, item: MenuItem): Boolean {
-        val selectedText = mSelection?.text ?: return super.performAction(id, item)
+        try {
+            val selectedText = mSelection?.text ?: return super.performAction(id, item)
 
-        return customDelegate.performAction(id, selectedText) || super.performAction(id, item)
+            return customDelegate.performAction(id, selectedText) || super.performAction(id, item)
+        } catch (e: SecurityException) {
+            return false
```

Comment:
I think after the gv bug is resolved we have to remove the catch, as it could hide other issues? If so, I think we should link the bug and add a comment to indicate that the catch should be removed after.

Reply:
Agreed.  I'll add a comment.  Thanks,